helm/robusta/templates/runner.yaml

```diff
@@ -31,6 +31,12 @@ spec:
             value: "true"
           - name: LOG_LEVEL
             value: {{ .Values.runner.log_level }}
+          - name: SECRET_SENTRY_DSN
+            valueFrom:
+              secretKeyRef:
+                name: secret-sentry
```

Comment:
I would name the secret a little more generic
like, robusta-runner-secret
We will probably add other secrets to it sometime in the future.

and because of the above, I would rename dsn to sentry-dsn or similar